coin/coin_domain/src/main/java/com/pavluyk/coin_domain/usecases/CoinsUseCase.kt

```diff
@@ -0,0 +1,24 @@
+package com.pavluyk.coin_domain.usecases
+
+import com.pavluyk.coin_domain.models.CoinModel
+import com.pavluyk.coin_domain.repositories.CoinMonitoringRepository
+import kotlinx.coroutines.flow.Flow
+
+class CoinsUseCase(private val coinMonitoringRepository: CoinMonitoringRepository) {
```

Comment:
this is some kind of bullshit.
Usecase is implementation of one (simple or more complex) action or intention. One action which can have/not input/result.

this kind of layer useless, so probably it should be 3 separate UseCases